From 9a3f54a8b05c75efe24ad248cf33bbf1acd4835c Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Mon, 30 Jul 2018 12:26:23 -0700
Subject: [PATCH] TRITON-568 CNAPI should stop using
 restify.authorizationParser() Reviewed by: Orlando Vazquez
 <orlando@joyent.com> Approved by: Orlando Vazquez <orlando@joyent.com>

---
 lib/server.js | 3 +--
 package.json  | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/lib/server.js b/lib/server.js
index aed3911..acd0644 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 var os = require('os');
@@ -52,7 +52,6 @@ function createServer(options) {
     });
 
     cnapi.use(restify.acceptParser(cnapi.acceptable));
-    cnapi.use(restify.authorizationParser());
     cnapi.use(restify.queryParser({
         mapParams: true,
         allowDots: false,
diff --git a/package.json b/package.json
index 0411506..223f3e1 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cnapi",
   "description": "SmartDataCenter Compute Node API",
-  "version": "1.11.1",
+  "version": "1.11.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
-- 
2.21.0

